apps/dashboard/src/components/workflow-editor/schema.ts

```diff
@@ -43,3 +45,68 @@ export const formSchema = z.object({
     }),
   }),
 });
+
+export type JSONSchema = WorkflowTestDataResponseDto['to'];
+
+export const buildDynamicFormSchema = ({
```

Comment:
The helper that builds the dynamic Zod schema based on JSONSchema returned from the API.
I was looking into the `npm` package that can do that, and I found one but it was generating the string for JS code that you needed to evaluate, so I resigned, [ref](https://github.com/StefanTerdell/json-schema-to-zod?tab=readme-ov-file#use-at-runtime).

Reply:
Might be useful in the future when we will be building controls form in step editor